languages-and-frameworks/svelte.html.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "Run a SvelteKit App"
+layout: language-and-framework-docs
+sitemap: false
+redirect_from: /docs/getting-started/sveltekit/
+---
+
+<% app_name = "hello-sveltekit" %>
+<%= partial "partials/intro", locals: { runtime: "SvelteKit", link: "https://kit.svelte.dev" } %>
+
+We'll be using the standard web application generated by [SvelteKit](https://kit.svelte.dev/). This is a bare-bones app with no database.
+
+## _Generate the SvelteKit app_
+
+We'll assume you have NodeJS installed already and can run `npm`. We recommend
+using `npm` over `yarn`, as `npm` is what SvelteKit [references in their documentation](https://kit.svelte.dev/docs/creating-a-project).
+
+Now let's create a new project by running `npm create`. This will scaffold a new project asking you if you'd like to set up some basic tooling such as TypeScript.
+
+```cmd
+npm create svelte@latest hello-sveltekit
+```
+```output
+Need to install the following packages:
+  create-svelte@5.0.2
+Ok to proceed? (y) y
+
+create-svelte version 5.0.2
+
+┌  Welcome to SvelteKit!
+│
+◇  Which Svelte app template?
+│  SvelteKit demo app
+│
+◇  Add type checking with TypeScript?
+│  Yes, using JavaScript with JSDoc comments
+│
+◇  Select additional options (use arrow keys/space bar)
+│  none
+│
+└  Your project is ready!
+
+...
+
+Next steps:
+  1: cd hello-sveltekit
+  2: npm install (or pnpm install, etc)
+  3: git init && git add -A && git commit -m "Initial commit" (optional)
+  4: npm run dev -- --open
+
+To close the dev server, hit Ctrl-C
+
+Stuck? Visit us at https://svelte.dev/chat
+...
+```
+
+Now, let's run `npm install` to install the necessary dependencies and proceed with configuring the adapter.
+
+## _Configuring the Adapter_
+
+Before you can deploy your SvelteKit app, you need to _adapt_ it for your deployment target. Adapters are small plugins that take the built app as input and generate output for deployment.
+
+In this example, to deploy your app on Fly.io, we'll install the Node Adapter. You can install it by executing the command `npm i -D @sveltejs/adapter-node` and then adding it to our `svelte.config.js` file. Here's all the code:
```

Comment:
When I ran this, I found that there was already a `svelte.config.js` file present.  I then mentally diffed what I saw there with what you provided below and came to the conclusion that all I needed to modify was the import line.  Let's not make people repeat that process, and instead tell them the one line they need to modify is the import statement.  Since the file is small, it is fine for the guide to show the resulting file.

Reply:
Fixed